src/tdastro/sources/periodic_variable_star.py

```diff
@@ -29,9 +29,9 @@ class PeriodicVariableStar(PeriodicSource, ABC):
         The distance to the source, in pc.
     """
 
-    def __init__(self, period, t0, **kwargs):
-        super().__init__(period, t0, **kwargs)
-        if not self.has_valid_param("distance"):
+    def __init__(self, period, **kwargs):
+        super().__init__(period, **kwargs)
+        if "distance" not in self.setters:
```

Comment:
Not an issue, but a question: why this change?

Reply:
That is actually a merge artifact from working on this at the same time I was working on the `has_valid_param` I sent you earlier today. I am changing it back to `has_valid_param` since that also does the `None` check.